dandi/metadata.py

```diff
@@ -302,6 +318,216 @@ def extract_field(field, metadata):
         return metadata.get(field, ...)
 
 
+nd_list = {
```

Comment:
is this to be the canonical source of this listing? 
I think we should keep that google spreadsheet by @bendichter for now and auto-convert it to .tsv or .json and carry as a file to be loaded here.  Dumping all these data structures which can change here makes harder to navigate the code IMHO

it seems the key here is identical to `neurodata_type`, so should not be "hardcoded" but mapped at load time to avoid duplication (the evil of large % of bugs)

Reply:
i don't know where this will live eventually. i also need to add identifiers to this and so it may move to dandischema (at least some pieces of it).